lib/src/soloud.dart

```diff
@@ -242,7 +246,6 @@ interface class SoLoud {
       // Initialize [SoLoudLoader]
       _loader.automaticCleanup = automaticCleanup;
 
-      // TODO(filip): The Loader is not compatible with web!
       await _loader.initialize();
```

Comment:
the `Loader` has been splitted into `dart:io` and `dart:html` conditional imports.